File: test-schemas-domain-classes-scala3/src/main/scala/testdomains/generic/edges/EdgeTypes.scala
package testdomains.generic.edges

object AnotherEdge {
  val Label = "another_edge"

}

class AnotherEdge(src_4762: flatgraph.GNode, dst_4762: flatgraph.GNode, subSeq_4862: Int, property_4862: Any)
    extends flatgraph.Edge(src_4762, dst_4762, 0, subSeq_4862, property_4862) {}

object ConnectedTo {
  val Label                        = "connected_to"
  val propertyName: Option[String] = Some("string_mandatory")
}

class ConnectedTo(src_4762: flatgraph.GNode, dst_4762: flatgraph.GNode, subSeq_4862: Int, property_4862: Any)
    extends flatgraph.Edge(src_4762, dst_4762, 1, subSeq_4862, property_4862) {
  override def propertyName: Option[String] = ConnectedTo.propertyName
}

File: test-schemas-domain-classes-scala3/src/main/scala/testdomains/generic/EdgeTypes.java
package testdomains.generic;

import java.util.HashSet;
import java.util.Set;

public class EdgeTypes {


public static final String ANOTHER_EDGE = "another_edge";


public static final String CONNECTED_TO = "connected_to";

public static Set<String> ALL = new HashSet<String>() {{
add(ANOTHER_EDGE);
add(CONNECTED_TO);
}};

}